validator/docs/schemas.md

```diff
@@ -0,0 +1,163 @@
+# Configuration Validator
+
+This module contains the logic needed to validate deployment files with a defined schema by Nestor.
+
+The deployment files define applications or projects.
+
+## Application Configuration
+
+In each file you will define the application that you want to deploy, there are few fields which are required to setup a new application.
+
+This is a basic configuration application with the minimum required fields.
+
+```yaml
+app: my-application   # Application's name
+is_enabled: false     # Is application deployment enabled
+git: git@microservice_repo # Git repository URL
+variables:            # Environment variables
```

Comment:
I would try to align the comments (same for all comments)
```suggestion
app: my-application        # Application's name
is_enabled: false          # Is application deployment enabled
git: git@microservice_repo # Git repository URL
variables:                 # Environment variables
```

Reply:
Also, you sometimes end up with a lot of information in some of the comments (not especially here but in the other examples too).
What do you think of using a table describing what is expected and then having the example ? 🙂